kyaml/setters/lookupkio.go

```diff
@@ -10,15 +10,18 @@ import (
 	"sigs.k8s.io/kustomize/kyaml/yaml"
 )
 
-var _ kio.Filter = &LookupSetters{}
+var _ kio.Filter = &LookupDeleteSetters{}
 
-// LookupSetters identifies setters for a collection of Resources
-type LookupSetters struct {
+// LookupDeleteSetters identifies setters for a collection of Resources
```

Comment:
Does it make sense to merge lookup and delete?  seems no. A test would make this more obvious one way or the other.

Reply:
This code path is for v1 setters which will be deprecated eventually when users migrate to v2 with this command. This command uses the lookup code path and deletes them while looking up rather than digging a new path as a whole. The existing tests(https://github.com/kubernetes-sigs/kustomize/blob/master/cmd/config/internal/commands/cmdlistsetters.go) ensure the current behavior and the new tests(pushed) ensure the new behavior.